docs/notes/2.22.x.md

```diff
@@ -68,6 +68,10 @@ Setting [the `orphan_files_behaviour = "ignore"` option](https://www.pantsbuild.
 
 ### Plugin API changes
 
+## Bugfixes
+
+- Sandboxing: Log output from sandboxes will contain anonymized paths instead of completely removing the sandbox prefix. (#20492)
```

Comment:
This sort of cross-cutting thing often goes in a dedicated "overall" section: https://github.com/pantsbuild/pants/blob/main/docs/notes/2.20.x.md#overall

Maybe you could add one after "new options system" and move it up there? (I personally find it easier to read release notes where the changes are grouped by area rather than by "size"/"type")

Also, I think this may soon (https://github.com/pantsbuild/pantsbuild.org/pull/184) be rendered in places that don't auto-link the GitHub issues:

```suggestion
- Sandboxing: Log output from sandboxes will contain anonymized paths instead of completely removing the sandbox prefix. ([#20492](https://github.com/pantsbuild/pants/pull/20492))
```

(I note that we haven't historically been linking the PRs/issues for everything, although maybe it'd be good to do more consistently.)

Reply:
> Also, I think this may soon (https://github.com/pantsbuild/pantsbuild.org/pull/184) be rendered in places that don't auto-link the GitHub issues:

Or, maybe the doc site renderer we use (docusaurus) can do auto-linking. I'm not sure, but it seems plausible 🤔

Reply:
Had the same thought about auto linking during doc site rendering.. :)

Reply:
> Or, maybe the doc site renderer we use (docusaurus) can do auto-linking. I'm not sure, but it seems plausible 

I did a quick look through their docs and source code, and I couldn't find any indication of auto-linking support, so we'd have to write a plugin (and/or contribute it upstream)... so, for now, let's just write the links manually to ensure they work everywhere?

I've filed https://github.com/pantsbuild/pantsbuild.org/issues/203